pkg/startup/v6/startup.go

```diff
@@ -71,6 +71,10 @@ func (s *startup) WriteFiles(ctx context.Context) error {
 			return err
 		}
 	}
+	// we also need openshift.local.volumes dir created before xfs quota code runs
```

Comment:
would prefer this in the v6 startup shell script, with an additional comment saying to remove after we're on the new VM image across the board